scipy/stats/stats.py

```diff
@@ -7350,9 +7350,9 @@ def rankdata(a, method='average', *, axis=None):
     >>> rankdata([[0, 2], [3, 2]]).reshape(2,2)
     array([[1. , 2.5],
           [4. , 2.5]])
-    >>> rankdata([[0, 2], [3, 2]], axis = 1)
-    array([[1., 2.],
-          [2., 1.]])
+    >>> rankdata([[0, 2, 2], [3, 2, 5]], axis=1)
+    array([[1. , 2.5, 2.5],
+           [2. , 1. , 3. ]])
```

Comment:
Are those spaces after the decimal points from the NumPy repr? I assume that's not formally pep8?

Reply:
Yes, that's the output generated by numpy, which is not something we should edit.